### PR TITLE
Fix importer

### PR DESCRIPTION
**Pull request by jwest60**
Fixed all plant tags that weren't strictly numeric and added an old tag column to any plants that had their tag replaced. This old tag will appear with the rest of the plants view.

image

### CODE REVIEW

**Comment by jwest60** [comment]
Updated files have been added to ~/data-files on the server.

File: app/Console/Commands/ImportFile.php [hunk 1]
```diff
@@ -183,10 +183,15 @@ protected function processLine($i, $line)
...
             $url,
+            $plant_old_tag,
         ] = $line;
         array_walk($line, 'trim');
         $username = strtolower($username);
 
+        if ($plant_old_tag) {
+          $this->info('Old tag: ' . $plant_old_tag);
+        }
+
```
Comment (almasaeed2010) [inline app/Console/Commands/ImportFile.php:194]:
```suggestion
```

File: app/Console/Commands/ImportFile.php [hunk 2]
```diff
@@ -241,12 +246,13 @@ protected function processLine($i, $line)
         $species = $this->getSpecies($species_name);
 
         // Find the plant. If does not exist, create one
-        $plant = Plant::firstOrCreate([
+        $plant = Plant::updateOrCreate([
```
Comment (almasaeed2010) [inline app/Console/Commands/ImportFile.php:249]: Any reason we need updateOrCreate instead of firstOrCreate?

Comment (jwest60) [reply app/Console/Commands/ImportFile.php:245]: It made more sense when I had `old_tag` in the second array before I had moved it up to the first. I can switch it back if it's not necessary

**Pull request merged**